### PR TITLE
[14.0][ADD] modules integración Mod322 con Prorata especial

### PR DESCRIPTION
**Pull request by etobella**
Replaces https://github.com/oxigensalud/odoo-addons/pull/136

### CODE REVIEW

File: l10n_es_aeat_mod322_special_prorate/static/description/icon.png
Comment (eantones) [inline l10n_es_aeat_mod322_special_prorate/static/description/icon.png:1]: Este logo debería cambiarse en todos los módulos de este PR

**Review by eantones — CHANGES_REQUESTED** [review]

File: l10n_es_aeat_mod322_special_prorate/static/description/icon.png
Comment (etobella) [reply l10n_es_aeat_mod322_special_prorate/static/description/icon.png:1]: Toda la razón

**Review by eantones — APPROVED** [review]

File: l10n_es_aeat_mod322_special_prorate_regularization_capital_asset_legacy/__manifest__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "version": "14.0.1.0.1",
     "category": "Accounting",
     "author": "NuoBiT Solutions, S.L.",
```
Comment (eantones) [inline l10n_es_aeat_mod322_special_prorate_regularization_capital_asset_legacy/__manifest__.py]: Change this please

**Review by eantones — CHANGES_REQUESTED** [review]
Also, please squash the following commits so that I can proceed with the merge:

-  [[FIX] adapt test-requirements](https://github.com/oxigensalud/odoo-community-addons/pull/1/commits/275d4450fa049746b946bb64e6afa8af8fe67fc0) and [fixup! [FIX] adapt test-requirements](https://github.com/oxigensalud/odoo-community-addons/pull/1/commits/52954c0af71fea2aeff3bb5d6c26c72f253ec5a6) 
- [[FIX] pylint: Adapt](https://github.com/oxigensalud/odoo-community-addons/pull/1/commits/f25d79593fed1cfc95ccf589aaeb07f6e70528bb) with each of the "[ADD]" commits

File: l10n_es_aeat_mod322_special_prorate_regularization_capital_asset_legacy/__manifest__.py (hunk 1 above)
Comment (etobella) [reply l10n_es_aeat_mod322_special_prorate_regularization_capital_asset_legacy/__manifest__.py]: Done!

**Review by eantones — APPROVED** [review]

**Pull request merged**